protocol/x/accountplus/keeper/keeper.go

```diff
@@ -144,7 +147,7 @@ func (k Keeper) GetAllAuthenticatorData(ctx sdk.Context) ([]types.AuthenticatorD
 	return accountAuthenticators, nil
 }
 
-func GetAccountPlusStateWithTimestampNonceDetails(
+func AccountStateFromTimestampNonceDetails(
```

Comment:
Nit: this is only used for newly initialized account state
```suggestion
func NewAccountStateFromTimestampNonceDetails(
```